pkg/types/libvirt/machinepool.go

```diff
@@ -13,6 +13,8 @@ type MachinePool struct {
 	// Image is the URL to the OS image.
 	// E.g. "http://aos-ostree.rhev-ci-vms.eng.rdu2.redhat.com/rhcos/images/cloud/latest/rhcos-qemu.qcow2.gz"
 	Image string `json:"image"`
+	// Metadata string describing the image
+	ImageDescription string `json:"imageDescription,omitempty"`
```

Comment:
`MachinePool` is for user-supplied input (although we supply defaults when the user doesn't care).  This string doesn't seem like an actionable input.